force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -79,17 +86,21 @@ public with sharing class ClassesObjectsMethods {
      * @param sentence The input sentence.
      * @return A string consisting of only unique words from the input sentence.
      */
-    public static String removeDuplicateWords(String sentence) {
-        // Split the sentence into words
-
-        // Create a set to store the unique words
-
+     // Split the sentence into words
+       // Create a set to store the unique words
         // Add each word to the set
-
-        // Join the unique words back into a sentence
-
-        // Return the sentence with unique words
-        return null; // Replace null with the variable you used to store the result
+            // Join the unique words back into a sentence
+             // Return the sentence with unique words
+    public static String removeDuplicateWords(String sentence) {
+        List<String> wordList = sentence.split(' ');
+        Set<String> wordSet = new Set<String>();
+        wordSet.addAll(wordList);
+        String newSentence = '';
+         for (String word2 : wordSet) {
+            newSentence += word2 + ' ';
+         }
+        String newSentenceTrimmed = newSentence.trim();
+        return newSentenceTrimmed; // Replace null with the variable you used to store the result
```

Comment:
Great job, Love that you added the List directly to the Set.  check out String.join for your return.